ext/opentelemetry-ext-http-requests/src/opentelemetry/ext/http_requests/__init__.py

```diff
@@ -77,6 +79,10 @@ def instrumented_request(self, method, url, *args, **kwargs):
             span.set_attribute("http.status_code", result.status_code)
             span.set_attribute("http.status_text", result.reason)
 
+            propagator.global_propagator().inject(
```

Comment:
Probably off-topic, but: You are injecting the headers after the request was sent here, and into the response headers instead of the request headers.

I also wonder if we should automatically fall back to `setter.__setitem__` if `not callable(setter)`. Requiring users to pass dunder methods is a bit arcane.

Reply:
sorry yes, good point. Will fix that.

defaulting to __setitem__ is a good idea. Just requires someone to write a wrapper object if that doesn't already exist. A bound callable isn't a bad idea either.

I'll file an issue and pick that up in another PR.